frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -371,6 +375,17 @@ export const getInitialValues = (
     internalImageValues = _.isEmpty(externalImageValues)
       ? getInternalImageInitialValues(_.get(appResources, 'editAppResource.data'))
       : {};
+    if (
+      _.isEmpty(externalImageValues) &&
+      !_.get(internalImageValues, 'imageStream.tag') &&
```

Comment:
Can we optional chaining here instead of get function?

Reply:
Chaining would look for `type` here as above we have `{}` in case `externalImageValues` is not empty so added `get` 

```
internalImageValues = _.isEmpty(externalImageValues)
      ? getInternalImageInitialValues(_.get(appResources, 'editAppResource.data'))
      : {};
```